azure-pipelines.yml

```diff
@@ -89,8 +81,41 @@ jobs:
         name: python_codestyle
         pytest: false
 
+# Run these only on cron
+- ${{ if or(eq(variables['Build.Reason'], 'Schedule'), eq(variables['Build.Reason'], 'Manual')) }}:
```

Comment:
I think we could not replicate the whole thing. Just put some of the envs list under this conditional.

Reply:
Good point.

Reply:
Fixed.